src/cli/cli_udp.cpp

```diff
@@ -136,14 +136,27 @@ otError UdpExample::ProcessSend(int argc, char *argv[])
 {
     otError       error;
     otMessageInfo messageInfo;
-    otMessage *   message = NULL;
-    int           curArg  = 0;
+    otMessage *   message              = NULL;
+    int           curArg               = 0;
+    unsigned long autoGenMessageLength = 0;
 
     memset(&messageInfo, 0, sizeof(messageInfo));
 
-    VerifyOrExit(argc == 1 || argc == 3, error = OT_ERROR_INVALID_ARGS);
+    VerifyOrExit(argc == 1 || argc == 3 || argc == 4, error = OT_ERROR_INVALID_ARGS);
 
-    if (argc == 3)
+    if (argc == 4)
+    {
+        if (strcmp(argv[curArg++], (const char *)"-s") == 0)
```

Comment:
nit: is the cast to `const char *` necessary?

Reply:
no, removed.